examples/kafka-test/functions/onTest.ts

```diff
@@ -0,0 +1,3 @@
+export default async function (event) {
+  console.log(event)
```

Comment:
Let's implement an echo functionality. Let's send back exactly the same message you receive. This will help you test the "send" functionality. After sending a message from the Confluent UI, you should see a message appear in your topic with the same details. Hopefully, it doesn't enter an endless loop 😄 If it does, you can just create another topic where the "echo" will be sent to.

```suggestion
  return {
    send: [{
      server: event.serverName,
      channel: event.channel,
      payload: event.payload,
      headers: event.headers,
    }]
  }
```